source/extensions/quic_listeners/quiche/envoy_quic_client_connection.cc

```diff
@@ -73,7 +73,7 @@ void EnvoyQuicClientConnection::processPacket(
 
 uint64_t EnvoyQuicClientConnection::maxPacketSize() const {
   // TODO(danzh) make this variable configurable to support jumbo frames.
-  return Network::MAX_UDP_PACKET_SIZE;
+  return 1500;
```

Comment:
again prefer having the variable.

Also can this now pull from config?

Reply:
re: variable, will fix.

re: pull from config, this is for upstream connections. I wasn't sure if we want different config for upstream like we do for udp_proxy? I wasn't sure where to configure that and figured we could add it later.